packages/eas-cli/src/credentials/manager/SelectEasConfigFromJson.ts

```diff
@@ -0,0 +1,30 @@
+import { EasConfig, EasJsonReader } from '@expo/eas-json';
+
+import { promptAsync } from '../../prompts';
+
+export class SelectEasConfigFromJson {
```

Comment:
```suggestion
export class SelectBuildProfileFromEasJson {
```
EasConfig was named  to be consisted with ExpoConfig when eas-json package was called @eas/config, we will probably need to change this. In most places variable of that type is named profile or buildProfile